readme.md

```diff
@@ -93,8 +97,8 @@ Your Answer:
 How is the concept of OAuth related to a valet key?
 
 Your Answer:
-```text
-
+```
+	the valet key is another key within the browser when you log into/onto a website.  That's your special key when you log in. Its basically to keep a session on that website.
 ```
```

Comment:
Valet keys allow you to give someone the keys to your car without full access (e.g. they can't open the trunk or glovebox). Similarly, OAuth allows you to share your account on a 3rd party provider (e.g. Facebook, Twitter, Google), without given the consumer app your password, or necessarily full access to your account, such as reading / modifying  your data, posting on your behalf, etc.